tests/test_networks.py

```diff
@@ -805,7 +805,6 @@ def test_address_valid(value, name, email):
         ('\u0020@example.com', None),
         ('\u001f@example.com', None),
         ('"@example.com', None),
-        ('\"@example.com', None),
```

Comment:
was this change on purpose? It seems to not be a formatting change

Reply:
Yeah, I removed this test case bc `ruff` was removing the `\`. This case and the one directly above are identical.

Reply:
I believe the string formatter removes unnecessary backslashes. (The backslash here doesn’t have an effect.)